terraform/aws/ec2-2.tf

```diff
@@ -0,0 +1,308 @@
+resource "aws_instance" "web_host" {
+  # ec2 have plain text secrets in user data
+  ami           = "${var.ami}"
+  instance_type = "t2.nano"
+
+  vpc_security_group_ids = [
+  "${aws_security_group.web-node.id}"]
+  subnet_id = "${aws_subnet.web_subnet.id}"
+  user_data = <<EOF
+#! /bin/bash
+sudo apt-get update
+sudo apt-get install -y apache2
+sudo systemctl start apache2
+sudo systemctl enable apache2
+export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMAAA
+export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMAAAKEY
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** AWS   
**Description:** AWS Secret Access Key   
**Severity:** CRITICAL   
**Match:** export AWS_SECRET_ACCESS_KEY=****************************************  
[This comment was created by Aqua Pipeline]